### ISSUE TITLE
Yoast compatibility producing errors in the console

### ISSUE THREAD

**Issue by andrejarh** [siteorigin/siteorigin-panels#762]
Hi,
I've tested the new version a bit and there seems to be some issues with the recently added YOAST compatibility.

When YOAST plugin is active, this error is produced in the console:
![image](https://user-images.githubusercontent.com/10053621/83610542-e0d97a00-a57f-11ea-9f43-0b29332c2bf5.png)

And when YOAST plugin isn't active, I see this error in the console:
![image](https://user-images.githubusercontent.com/10053621/83610482-c8695f80-a57f-11ea-9bb8-b62e9fb99a4d.png)

This are my debug settings in wp-config.php:
```
define('WP_DEBUG', true);
ini_set('log_errors',TRUE);
ini_set('error_reporting', E_ALL);
ini_set('error_log', dirname(__FILE__) . '/error_log.txt');
ini_set('display_errors',1);
ini_set('display_startup_errors',1);
error_reporting(-1);
// gutenberg 
define('SCRIPT_DEBUG', true);
```

**Comment by gregpriday** [comment]
Thanks for the report @andrejarh - I think I can see what's going on. Our Yoast compat probably just needs a wrapping DIV, and a better check for YoastSEO. I'll take a look and fix this up.

**Comment by andrejarh** [comment]
Hi @Misplon 
I've installed a demo WP instance at https://siteorigin.orion.si/ 

If you edit the page, you'll notice the error in the console. Login credentials have been sent to SiteOrigin support@ mail.

Today, I was only able to reproduce the error when YOAST isn't active. If I can reproduce the error when YOAST is active, I'll write down the steps.

**Comment by Misplon** [comment]
Awesome, thanks for setting up a demo for us! Alex has taken a look and will reply shortly.

**Comment by AlexGStapleton** [comment]
Hi andrejarh ,

Thanks mate. I was able to replicate the second issue on that site, and I wasn't able to replicate the first issue either.

The second issue was actually resolved in #767 as I came across the same issue when adding Rank Math support. I didn't know about this issue at the time or I would have mentioned something.

**Comment by Misplon** [comment]
Hi @andrejarh 

I'm going to close this issue for the time being. If you're able to replicate the issue when Yoast IS active, please, let us know and we'll re-open.

Thanks for your help.
